Frameworks/Foundation/NSKeyedArchiver.mm

```diff
@@ -65,12 +66,7 @@ static inline void _checkKey(NSString* aKey, NSMutableDictionary* _enc) {
 * Make a dictionary referring to the object at ref in the array of all objects.
 */
 static id makeReference(unsigned ref) {
```

Comment:
>ref [](start = 33, length = 3)

nit: ref is a special word in some languages and can confuse some of us, could we rename this?